### PR TITLE
Configure & test Logentries Fastly integration

### PR DESCRIPTION
**Pull request by bart613**
As `tf_fastly_frontend` now has got Logentries integration, we need to pass releavent variables (`le_logset_id`) into that module.

Also, add unittest to make sure Logentries configuration is available as a result of this module.  Updated README too.

This PR is part of work in [JIRA PLAT-1030](https://workflow.mergermarket.com/browse/PLAT-1030)

### CODE REVIEW

File: fastly.tf [hunk 1]
```diff
@@ -1,9 +1,10 @@
 module "fastly" {
-  source = "github.com/mergermarket/tf_fastly_frontend"
+  source = "git::https://github.com/mergermarket/tf_fastly_frontend.git"
```
Comment (grahamlyons) [inline fastly.tf]: Any reason we've added this stuff on to the source?

Comment (bart613) [inline fastly.tf]: @grahamlyons yes - this format of source lets you easy add branch / tag you want to pin to

Comment (grahamlyons) [inline fastly.tf]: You can still add `?ref=<ref>` to the plain `github.com/...` one.

**Comment by bart613** [comment]
That wouldn't work for me :/

File: fastly.tf (hunk 1 above)
Comment (bart613) [inline fastly.tf]: I couldn't get that format to work...

**Pull request merged**